.docker/source/Dockerfile

```
@@ -11,20 +11,13 @@ RUN \
     cd src && \
     #
     # Download moveit source so that we can get necessary dependencies
-    wstool init . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init . https://raw.githubusercontent.com/PickNikRobotics/moveit2/moveit2-ci/moveit.rosinstall && \
```

To be applied before merging:
```suggestion
    wstool init . https://raw.githubusercontent.com/ros-planning/moveit2/${ROS_DISTRO}-devel/moveit.rosinstall && \
```